DataProcessor/DataUnpacker.cpp

```diff
@@ -164,6 +164,6 @@ void DataUnpacker::eng_dash_connection(bool state) {
 }
 
 bool DataUnpacker::checkRestartEnable() {
-    return battery_eStop || driver_eStop || external_eStop || imd_status || !door || crash || mcu_check || restart_enable;
+    return driver_eStop || external_eStop || imd_status || !door || crash || mcu_check || restart_enable || discharge_enable;
```

Comment:
```suggestion
    return driver_eStop || external_eStop || imd_status || door || crash || mcu_check || restart_enable || discharge_enable;
```
`door`'s error state changed to 1

Reply:
Please add a TODO comment to implement `mcu_hv_en` error checking here as well (don't implement it quite yet. This should be done at the same time as setting `mcu_hv_en` in the byte array when the user taps the screen to restart the car)